roqoqo/src/operations/mod.rs

```diff
@@ -813,37 +813,37 @@ pub trait OperateMultiQubitGate:
 }
 
 /// Marker trait to show that some operation has been implemented in roqoqo 1.1.0
-pub(crate) trait ImplementedIn1point1: Operate {}
+pub trait ImplementedIn1point1: Operate {}
```

Comment:
Was that change required by linting?

Reply:
`rustc` warned it was never used, I'll revert and add a `#[allow(dead_code)]`